docs/orchestrating-elastic-stack-applications/beat.asciidoc

```diff
@@ -0,0 +1,377 @@
+:page_id: beat
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Run Beats on ECK
+
+This section describes how to configure and deploy Beats with ECK.
+
+* <<{p}-beat-quickstart,Quickstart>>
+* <<{p}-beat-configuration,Configuration>>
+* <<{p}-beat-troubleshooting,Troubleshooting>>
+
+[id="{p}-beat-quickstart"]
+== Quickstart
+
+. Apply the following specification to deploy Filebeat and collect the logs of all containers running in the Kubernetes cluster. ECK automatically configures the secured connection to an Elasticsearch cluster named `quickstart`, created in the link:k8s-quickstart.html[Elasticsearch quickstart].
++
+[source,yaml,subs="attributes,+macros"]
+----
+cat $$<<$$EOF | kubectl apply -f -
+apiVersion: beat.k8s.elastic.co/v1beta1
+kind: Beat
+metadata:
+  name: quickstart
+spec:
+  type: filebeat
+  version: {version}
+  elasticsearchRef:
+    name: quickstart
+  config:
+    filebeat.inputs:
+    - type: container
+      paths:
+      - /var/log/containers/*.log
+  daemonSet:
+    podTemplate:
+      spec:
+        dnsPolicy: ClusterFirstWithHostNet
+        hostNetwork: true
+        securityContext:
+          runAsUser: 0
+        containers:
+        - name: filebeat
+          volumeMounts:
+          - name: varlogcontainers
+            mountPath: /var/log/containers
+          - name: varlogpods
+            mountPath: /var/log/pods
+          - name: varlibdockercontainers
+            mountPath: /var/lib/docker/containers
+        volumes:
+        - name: varlogcontainers
+          hostPath:
+            path: /var/log/containers
+        - name: varlogpods
+          hostPath:
+            path: /var/log/pods
+        - name: varlibdockercontainers
+          hostPath:
+            path: /var/lib/docker/containers
+EOF
+----
+
+. Monitor Beats
```

Comment:
I wonder if there's a better verb here than "monitor". "Inspect Beat status" maybe?

Reply:
This is following the other quickstarts we have, we've used Monitor in this section so far.